pkg/network/usm/monitor_test.go

```diff
@@ -922,30 +931,38 @@ func getClientsArray(t *testing.T, size int) []*nethttp.Client {
 	return res
 }
 
-func startH2CServer(t *testing.T) {
-	t.Helper()
-
+func startH2CServer(address string, isTLS bool) (func(), error) {
 	srv := &nethttp.Server{
-		Addr: http2SrvPortStr,
+		Addr: authority,
```

Comment:
why did you change it to authority?

Reply:
to ensure we're tide to the `lo` interface, and not to all interfaces